.env.example

```diff
@@ -1,3 +1,11 @@
+# Random generation ===========================================================
+# If you change this, check the collision probability https://zelark.github.io/nano-id-cc/
+NORAY_RANDOM_ID_LENGTH=21 # For 10 id/hour, 15 trillion years
+# NORAY_RANDOM_ID_LENGTH=4 # For 10 id/hour, 2 days
+# NORAY_RANDOM_ID_LENGTH=5 # For 10 id/hour, 19 days
+# NORAY_RANDOM_ID_LENGTH=6 # For 10 id/hour, 155 days
+NORAY_RANDOM_ID_LETTERS=useandom-26T198340PX75pxJACKVERYMINDBUSHWOLF_GQZbfghjklqvwyzrict
```

Comment:
Just curious, what's the reason behind the letter ordering?

Reply:
This is just the default the packages has, and it's like this for some compression optimisation.
Comment from: https://github.com/ai/nanoid/blob/main/url-alphabet/index.js#L1C1-L3C34
// This alphabet uses `A-Za-z0-9_-` symbols.
// The order of characters is optimized for better gzip and brotli compression.
// Same as in non-secure/index.js

Reply:
Fair enough!